pkg/image/api/helper.go

```diff
@@ -7,8 +7,8 @@ import (
 	"github.com/fsouza/go-dockerclient"
 )
 
-// dockerDefaultNamespace is the value for namespace when a single segment name is provided.
-const dockerDefaultNamespace = "library"
+// DockerDefaultNamespace is the value for namespace when a single segment name is provided.
+const DockerDefaultNamespace = "library"
```

Comment:
You exposed that constant, but you're not using it anywhere. Are you planning to use it inside `validation.go`? Maybe it'll make more sense to move it to `api` package then?